src/service/admin-file-service.js

```diff
@@ -66,15 +72,69 @@ module.exports = function (config) {
         /**
          * Get a directory listing, or contents of a file
          * @param  {String} `filePath`   Path to the file
-         * @param  {String} `folderType` One of Model|Static|Node
          * @param  {Object} `options` (Optional) Overrides for configuration options.
          */
-        getContents: function (filePath, folderType, options) {
-            var path = folderType + '/' + filePath;
+        getContents: function (filePath, options) {
+            var path = serviceOptions.folderType + '/' + filePath;
```

Comment:
rather than do the concatenation here I _think_ you can just do this as part of the http options, see https://github.com/forio/epicenter-js-libs/blob/master/src/service/data-api-service.js#L77